### PR TITLE
jackett indexers as of 10e6536

### PR DESCRIPTION
**Pull request by bakerboy448**

### CODE REVIEW

File: definitions/v1/mircrew.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    autologin: on
+    viewonline: on
+  error:
+    - selector: div.error
+  test:
+    path: index.php
+    selector: a[href^="./ucp.php?mode=logout&sid="]
+
+download:
+  selectors:
```
Comment (Qstick) [inline definitions/v1/mircrew.yml:91]: Download / Selectors is not valid in v1, this needs reworked and this version added to v2

**Review by Qstick — CHANGES_REQUESTED** [review]

File: definitions/v1/mircrew.yml (hunk 1 above)
Comment (bakerboy448) [reply definitions/v1/mircrew.yml]: mircrew moved to v2;  will not be backported to v1 / will not be supported until https://github.com/Prowlarr/Prowlarr/pull/310 lands; similar to other indexers jackett has added that use download selectors.

Comment (Qstick) [reply definitions/v1/mircrew.yml]: You COULD modify to work with v1 if you loved the users :P

**Review by Qstick — APPROVED** [review]

File: definitions/v1/mircrew.yml (hunk 1 above)
Comment (bakerboy448) [reply definitions/v1/mircrew.yml]: We have not done it for any other indexer; I see no reason to start now.

**Pull request merged**